src/core/backbone_router/leader.cpp

```diff
@@ -87,47 +83,56 @@ otError Leader::GetServiceId(uint8_t &aServiceId) const
     return error;
 }
 
+#if (OPENTHREAD_CONFIG_LOG_LEVEL >= OT_LOG_LEVEL_INFO) && (OPENTHREAD_CONFIG_LOG_BBR == 1)
 void Leader::LogBackboneRouterPrimary(State aState, const BackboneRouterConfig &aConfig) const
 {
     OT_UNUSED_VARIABLE(aConfig);
 
-    otLogInfoNetData("BBR state %s", StateToString(aState));
+    otLogInfoBbr("PBBR state: %s", StateToString(aState));
 
     if (aState != kStateRemoved && aState != kStateNone)
     {
-        otLogInfoNetData("Rloc16: 0x%4X, seqno: %d, delay: %d, timeout %d", aConfig.mServer16, aConfig.mSequenceNumber,
-                         aConfig.mReregistrationDelay, aConfig.mMlrTimeout);
+        otLogInfoBbr("Rloc16: 0x%4X, seqno: %d, delay: %d, timeout %d", aConfig.mServer16, aConfig.mSequenceNumber,
```

Comment:
merge logs to one line ?

Reply:
This PR serves only for logging update, I'd prefer not to introduce unrelated changes into this one.

Besides, regarding the comment, there is no valid dataset when the state is `kStateRemoved` or `kStateNone`,  one line for all scenarios would make it not easy to read the code in my view.

Any suggestion to achieve one line log? @simonlingoogle

Reply:
We can simply use `if else` with 2 different lines of `otLogInfoBbr`, but only one will be executed at runtime.

Reply:
It is an option. No strong opinion. Anyway feel free to submit a PR for it if you prefer this way. Thanks.

Will mark this as resolved as it is not related to this PR.